erica_app/tests/request_processing/test_eric_mapper.py

```diff
@@ -0,0 +1,725 @@
+
+            confirm_complete_correct=True,
+            confirm_send=True
+        )
+
+
+class TestEstDataPersonAFahrkostenPauschale:
+
+    def test_if_person_a_has_pflegegrad_and_requests_fahrkostenpauschale_then_set_higher_fahrkostenpauschale_for_all_disability_degrees(self, standard_est_input_data):
+        disability_degrees = [20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 100]
+        standard_est_input_data.person_a_has_pflegegrad = True
+        standard_est_input_data.person_a_requests_fahrkostenpauschale = True
+
+        for disability_degree in disability_degrees:
+            standard_est_input_data.person_a_disability_degree = disability_degree
+
+            resulting_input_data = EstEricMapping.parse_obj(standard_est_input_data)
+
+            assert resulting_input_data.person_a_fahrkostenpauschale_has_merkzeichen_bl_tbl_h_ag_pflegegrad is True
+            assert resulting_input_data.person_a_fahrkostenpauschale_has_merkzeichen_g_and_degree_70_degree_80 is None
+
+    def test_if_person_a_has_pflegegrad_and_does_not_want_fahrkostenpauschale_then_set_no_fahrkostenpauschale_for_all_disability_degrees(self, standard_est_input_data):
+        disability_degrees = [20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 100]
+        standard_est_input_data.person_a_has_pflegegrad = True
+        standard_est_input_data.person_a_requests_fahrkostenpauschale = False
+
+        for disability_degree in disability_degrees:
+            standard_est_input_data.person_a_disability_degree = disability_degree
+
+            resulting_input_data = EstEricMapping.parse_obj(standard_est_input_data)
+
+            assert resulting_input_data.person_a_fahrkostenpauschale_has_merkzeichen_bl_tbl_h_ag_pflegegrad is None
+            assert resulting_input_data.person_a_fahrkostenpauschale_has_merkzeichen_g_and_degree_70_degree_80 is None
+
+    def test_if_person_a_has_merkzeichen_bl_and_requests_fahrkostenpauschale_then_set_higher_fahrkostenpauschale_for_all_disability_degrees(self, standard_est_input_data):
+        disability_degrees = [20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 100]
+        standard_est_input_data.person_a_has_merkzeichen_bl = True
+        standard_est_input_data.person_a_requests_fahrkostenpauschale = True
+
+        for disability_degree in disability_degrees:
+            standard_est_input_data.person_a_disability_degree = disability_degree
+
+            resulting_input_data = EstEricMapping.parse_obj(standard_est_input_data)
+
+            assert resulting_input_data.person_a_fahrkostenpauschale_has_merkzeichen_bl_tbl_h_ag_pflegegrad is True
+            assert resulting_input_data.person_a_fahrkostenpauschale_has_merkzeichen_g_and_degree_70_degree_80 is None
+
+    def test_if_person_a_has_merkzeichen_bl_and_does_not_want_fahrkostenpauschale_then_set_no_fahrkostenpauschale_for_all_disability_degrees(self, standard_est_input_data):
```

Comment:
is the same test like: `test_if_person_a_has_pflegegrad_and_does_not_want_fahrkostenpauschale_then_set_no_fahrkostenpauschale_for_all_disability_degrees`

Reply:
I think it is different as one has the pflegegrad set and the other one hast the merkzeichen_bl set. Or do you mean that those are the same equivalence classes?

Reply:
Oh snap, I'm sry :sweat_smile: